x-pack/docs/en/security/securing-communications/security-basic-setup-https.asciidoc

```diff
@@ -486,8 +488,8 @@ separate monitoring cluster prevents production cluster outages from impacting
 your ability to access your monitoring data. It also prevents monitoring
 activities from impacting the performance of your production cluster.
 
-. From the directory where you installed Elasticsearch, navigate to the
-`/kibana` directory that you created when <<encrypt-http-communication,encrypting HTTP client communications for {es}>>.
+. On any single node, navigate to the `/kibana` directory that you created when
```

Comment:
Should this be `On the node on which you generated the HTTP certificates` ? (or neater wording to that effect)

I don't think we would assume that they copied the `kibana` directory to every node (though they might have)